components-core/src/main/java/com/adyen/checkout/components/model/connection/OrderPaymentMethod.kt

```diff
@@ -38,7 +38,7 @@ data class OrderPaymentMethod(
         @JvmField
         val CREATOR: Parcelable.Creator<OrderPaymentMethod> = Creator(OrderPaymentMethod::class.java)
 
-        @JvmStatic
+        @JvmField
```

Comment:
Were you able to double check if other classes have the same issue? If not we can open a separate ticket for it

Reply:
I opened a separate ticket for it